test/rqg/mzcompose.py

```diff
@@ -149,7 +178,12 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
     parser.add_argument(
         "--duration",
         type=int,
-        help="Run the Workload for the specifid time in seconds",
+        help="Run the Workload for the specified time in seconds",
+    )
+    parser.add_argument(
```

Comment:
Added `--queries` CLI option for directly providing a the number of queries while developing / debugging.